A local extern declares GfGfxLoader_LoadCharData void; the definition
(gf_gfx_loader.c:8) returns u32. Harmless on ARM, a trapping
signature-mismatch call on wasm. The declaration takes the real return type.

--- a/ov21_02254A6C.c
+++ b/ov21_02254A6C.c
@@ -28,7 +28,7 @@
 u8 const ov21_02254D90[] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x00, 0x0E, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
 
 extern void InitBgFromTemplate(u32 param0, u32 param1, void *param2, u32 param3);
-extern void GfGfxLoader_LoadCharData(u32 param0, u32 param1, u32 param2, u32 param3, u32 param4, u32 param5, u32 param6, u32 param7);
+extern u32 GfGfxLoader_LoadCharData(u32 param0, u32 param1, u32 param2, u32 param3, u32 param4, u32 param5, u32 param6, u32 param7);
 extern void GfGfxLoader_LoadScrnData(u32 param0, u32 param1, u32 param2, u32 param3, u32 param4, u32 param5, u32 param6, u32 param7);
 extern void ov20_02252D7C(u32 param0, u32 param1);
 extern void BgCommitTilemapBufferToVram(u32 param0, u32 param1);
